src/services/AccessTokenRequestProcessor.ts

```diff
@@ -117,8 +117,9 @@ export class AccessTokenRequestProcessor {
 					}),
 				};
 			} else {
-				this.logger.warn(`Session is in the wrong state: ${session.authSessionState}, expected state should be ${AuthSessionState.F2F_AUTH_CODE_ISSUED}`, { messageCode: MessageCodes.INCORRECT_SESSION_STATE });
-				return Response(HttpCodesEnum.UNAUTHORIZED, `Session is in the wrong state: ${session.authSessionState}`);
+				this.metrics.addMetric("AccessToken_error_user_state_incorrect", MetricUnits.Count, 1);
+				this.logger.warn( { message: `Session for journey ${session?.clientSessionId} is in the wrong Auth state: expected state - ${AuthSessionState.F2F_AUTH_CODE_ISSUED}, actual state - ${session.authSessionState}` }, { messageCode: MessageCodes.INCORRECT_SESSION_STATE });
```

Comment:
Does this need `{ message: `Session for journey` the first string of the log method should be message by default